api/pagination.go

```diff
@@ -9,5 +9,7 @@ type PaginationResponse struct {
 	Page  int `json:"page,omitempty"`
 	Limit int `json:"limit,omitempty"`
 
-	//TODO: add some indication of number of records/pages or if a next page exists
+	TotalCount int  `json:"total_count,omitempty"`
```

Comment:
We have a `count` field that is returned by the API today – should that be removed in this change?

Suggestion on cleaner naming:

`count` for this field and `pages` below